int-config.toml

```diff
@@ -104,7 +112,17 @@ lockfiles = ["./integration/data/lockfile/gobinary"]
 
 [servers.jar]
 type = "pseudo"
-lockfiles = ["./integration/data/lockfile/test.jar"]
```

Comment:
If you have prepared log4j-core-2.13.0.jar instead of test.jar, please delete test.jar that is not used in integration test.

Reply:
Nice catch! removed at 60d274b